mne_bids/write.py

```diff
@@ -1072,4 +1089,60 @@ def write_anat(bids_root, subject, t1w, session=None, acquisition=None,
         fname = t1w_basename.replace('.nii.gz', '.json')
         _write_json(fname, t1w_json, overwrite, verbose)
 
+        if deface is not None:
+            # x: L/R L+, y: S/I I+, z: A/P A+
```

Comment:
still missing where this is copied from

Reply:
I figured that out from messing with the image... that is the directionality in mri voxel space

Reply:
I thought the code was copied from FieldTrip somewhere, no?

Reply:
No, just useful for space transformations if this has to be done again